include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -109,7 +109,7 @@ struct layout_stride {
 
     //----------------------------------------------------------------------------
 
-    using __strides_storage_t = std::array<index_type, extents_type::rank()>;
+    using __strides_storage_t = detail::possibly_empty_array<index_type, extents_type::rank()>;
```

Comment:
get rid of use of std::array, for GPUs etc. actually this is what the LLVM implementation does too, will make the class empty for rank-0 with clang (where std:;array actually contains a char).